.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,216 @@
+        run: npm ci
+
+      - name: Get latest release tag
+        id: latest_release
+        run: |
+          latest_tag=$(gh release view --json tagName | jq -r '.tagName')
+          echo "latest_tag=$latest_tag" >> $GITHUB_OUTPUT
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Get LS versions and check if `langServer.version` needs update
+        id: ls_version
+        run: |
+          # Version of tofu-ls
+          current_ls_version=$(jq -r '.langServer.version' package.json)
+
+          # Latest tofu-ls version
+          latest_ls_version=$(gh release view -R opentofu/tofu-ls --json tagName | jq -r '.tagName')
+
+          # Check if we need to update the language server version and set it in output
+          # We are updating ls version if specified non-empty version.
+          ls_version="${{ inputs.ls_version }}"
+          # Flag to indicate this workflow will need to update the ls version
+          updates_ls_version=false
+          if [ "$ls_version" != "" ]; then
+            # If specified version is 'latest' and the current version is the latest, we don't update the version.
+            if [ "$ls_version" == "latest" ] && [ "$current_ls_version" == "$latest_ls_version" ]; then
+              ls_version=$latest_ls_version
+            else
+              # $ls_version in this case is already set from input
+              updates_ls_version=true
+              echo "ls_version_changed=true" >> $GITHUB_OUTPUT
+            fi
+          else
+            ls_version=$current_ls_version
+          fi
+
+          # Even if the ls version wasn't set during the workflow run it still can be different from the one in the last release of the extension
+          # We can check if that is the case
+          if [[ $updates_ls_version == false ]]; then
+            # We can get the langServer set in the last release using `git show` and latest tag we obtained in the last set
+            previous_version=$(git show ${{ steps.latest_release.outputs.latest_tag }}:package.json | jq -r '.langServer.version')
+            # We add a new flag to indicate the case, when tofu-ls version changed but not necessarily by the workflow and we can prepend it in release notes
+            if [[ "$previous_version" != "$current_ls_version" ]]; then
+              echo "ls_version_changed=true" >> $GITHUB_OUTPUT
+            fi
+          fi
+
+          # This output indicates what the ls version will be after it is updated in the next step
+          # Hence, it is set to the current version if no update occurs
+          echo "ls_version=$ls_version" >> $GITHUB_OUTPUT
+          echo "updates_ls_version=$updates_ls_version" >> $GITHUB_OUTPUT
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get current version and update it to the next one
+        id: version
+        run: |
+          # Get current version of the extension
+          current_version=$(jq -r '.version' package.json)
```

Comment:
This should be the same as the latest tag, right? With the value retried in step "- name: Get latest release tag".
Just checking, since I am not familiar with the rules around this repo.